lesson01/ash01-02/script.js

```diff
@@ -1,12 +1,69 @@
+// jQuery Doc Ready
 $(function () {
+  // Setting up click event on first button
   $("#buttonOne").click(function (){
+    // Declaring variables for this function
     const numSet = [5, 10, 15];
     let sum = 0;
+    // Collecting input entered by user
     const num = parseInt($("#inputOne").val());
+    // Setting up reduce function; combines all number in the array into one
     let sum2 = numSet.reduce(function(sum, value){
       return sum + value;
     });
+    // Adding the user's number to the total from the reduce function
     sum2 = sum2 + num;
-      alert(sum2);
+      // Displaying the final total on the web page
+      document.getElementById('answerOne').innerHTML = (sum2);
+  });
+});
+
+
+// jQuery Doc Ready
+$(function () {
```

Comment:
While each of the buttons should have their own click event listener, they can all be inside the same document ready event listener. It's not causing any problems to have multiple document ready listeners, it's just not necessary.